CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+
+# Contribution Guidelines
+
+In this project we need to find a compromise between pragmatism and engineering rigor.
+While contributions from everybody are encouraged mistakes might impact people's health.
```

Comment:
Contributions to this project from novices, beginners, intermediary, and advanced developers are welcome.
However, we need to find a compromise between pragmatism and engineering rigor because mistakes might impact people's health. Also [software entropy](https://pragprog.com/the-pragmatic-programmer/extracts/software-entropy) could bring the project to hold.

So the guidelines below are supposed to help to keep the code in this project in a high-quality shape. If you think that they should be improved in some way or the other, please create a ticket with your suggestion [here](https://github.com/ksch-workflows/ksch-workflows/issues/new).